cli/tests/testdata/run/stdout_write_all.out

```diff
@@ -0,0 +1,100 @@
+Hello, world!
```

Comment:
I presume that a non-synchronous write order might end up with the output file containing eg. `HHHHHHeeeeellllloooo,,,,,` etc.

Nevertheless, this out file is kind of pretty terrible in terms of reading this giving some understanding about what the output means. Would it be possible to add an index counter at the end of the line? So then this would be:
```
Hello, world! 1
Hello, world! 2
```
etc.

Reply:
I'll add it.